### PR TITLE
pgAdmin3: fix the port

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

See commits.

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
pgadmin

File: aqua/pgAdmin3/Portfile [hunk 1]
```diff
@@ -30,34 +31,44 @@ checksums           rmd160  48e732089cd1879f99d1894229a9cc23ed3c0c44 \
...
+configure.args      --enable-appbundle \
+                    --with-arch-${configure.build_arch} \
+                    --with-libgcrypt=${prefix} \
                     --with-libxml2=${prefix} \
                     --with-libxslt=${prefix} \
-                    --with-arch-x86_64 \
-                    --enable-appbundle \
-                    --with-libgcrypt=${prefix} \
-                    --with-openssl=no
+                    --with-openssl=no \
```
Comment (reneeotten) [inline aqua/pgAdmin3/Portfile:55]: if the dependency is there anyway, why not set this to "yes"?

File: aqua/pgAdmin3/Portfile [hunk 2]
```diff
@@ -30,34 +31,44 @@ checksums           rmd160  48e732089cd1879f99d1894229a9cc23ed3c0c44 \
...
 # For unknown reasons pgAdmin3.app/Contents/MacOS/pgAdmin3 still links to
-# libcrypto.1.1.dylib, so the openssl dependency is needed even if pgAdmin3
+# libcrypto.dylib, so the openssl dependency is needed even if pgAdmin3
 # is built with libgcrypt.
 depends_lib         path:lib/pkgconfig/icu-uc.pc:icu \
-                    path:lib/libssl.dylib:openssl \
                     port:libgcrypt \
                     port:libxml2 \
                     port:libxslt \
+                    path:lib/libssl.dylib:openssl \
```
Comment (reneeotten) [inline aqua/pgAdmin3/Portfile:47]: can this use the "openssl" PortGroup?

File: aqua/pgAdmin3/Portfile (hunk 1 above)
Comment (barracuda156) [reply aqua/pgAdmin3/Portfile:55]: I have no objections, I just do not know what that gonna entail, so kept non-breaking choices as they were.

File: aqua/pgAdmin3/Portfile (hunk 2 above)
Comment (barracuda156) [reply aqua/pgAdmin3/Portfile:47]: Original intent seems to have been to declare a dependency only due to supposedly unnecessary linking.

If we turn support for OpenSSL on, then PG is a better choice, of course.

**Pull request merged**